Lab08_MonteCarlo.Rmd

```diff
@@ -132,10 +157,20 @@ sens_j
 ```
 ## Task 5: interpreting sensitivity
 
-* Which parameters causes photosynthetic rate to increase? Which cause it to decrease?
+The plots we have provided show how the output of the FvCB function (i.e. the prediction of assimilation) changes when we **adjust only ONE parameter at a time**. In some cases changing the value of a parameter by only a small increment can have a dramatic change in the model prediction, while in other cases, you may see little to no response at all. This isn't the first time you've encountered this concept. In lab 6 when you created a shiny app with sliders for each variable, your task was to interactively **adjust the values of each parameters one at a time** and visually assess how it affected the performance of the model until you found a curve that passed through the points. Sometimes changing a variable shifted the curve over, sometimes it changed the shape of the curve. That behavior can be explained by the model's **sensitivity** to the respective parameters. In this lab we are finally quantifying what you were merely observing them. 
```

Comment:
Betsy, this text is WRONG, please fix and make sure this version is not sent to students. We just did a global sensitivity analysis, NOT a one-at-a-time analysis.